test/jdk/java/awt/print/PrinterJob/SecurityDialogTest.java

```diff
@@ -39,11 +39,13 @@
  * @test
  * @bug 4937672 5100706 6252456
  * @key printer
+ * @summary tests native and cross-platform page and print dialog
+ *          when security manager is installed
```

Comment:
Verifies "Print to file" option is disable if reading/writing files is not allowed by Security Manager.

I believe this type of summary explains better the purpose of the test, and the purpose is to ensure "Print to file" is disabled in writing to files is not allowed, according to the instructions of in the test.